internal/display/apis.go

```diff
@@ -16,16 +17,18 @@ type apiView struct {
 	Name       string
 	Identifier string
 	Scopes     string
+	TokenLifetime int
+	AllowOfflineAccess bool
 
 	raw interface{}
 }
 
 func (v *apiView) AsTableHeader() []string {
-	return []string{"ID", "Name", "Identifier", "Scopes"}
+	return []string{"ID", "Name", "Identifier", "Scopes", "TokenLifetime", "AllowOfflineAccess"}
```

Comment:
Let's keep these new fields off the table view, as there's already enough info in there.